### ISSUE TITLE
The reproducibility metrics do not match the results in the paper

### ISSUE THREAD

**Issue by 3232731490** [chanchanchan97/ICAFusion#44]
When I replicated the experiment on A40, the indicator did not reach the effect in the author's paper, and the map50 was only about 0.75, did the author use the pre-training weight of yolov5-l when running the experiment?

**Comment by 3232731490** [comment]
> When I replicated the experiment on A40, the indicator did not reach the effect in the author's paper, and the map50 was only about 0.75, did the author use the pre-training weight of yolov5-l when running the experiment?

After loading the Yolov5L pre-training weights and then training, the MAP50 indicator directly reaches about 0.81, which is higher than the index in the paper.

**Comment by buyuxixi** [comment]
请问你用的是这个配置吗？
 parser.add_argument('--weights', type=str, default='/root/autodl-tmp/ICAF\models\yolov5l.yaml', help='initial weights path')
 parser.add_argument('--cfg', type=str, default='./models/transformer/yolov5l_Transfusion_FLIR.yaml', help='model.yaml path')
 parser.add_argument('--data', type=str, default='/root/autodl-tmp/ICAF/data/multispectral/FLIR-align-3class.yaml', help='data.yaml path')
 我为啥跑不出来0.81

**Comment by chanchanchan97** [comment]
> When I replicated the experiment on A40, the indicator did not reach the effect in the author's paper, and the map50 was only about 0.75, did the author use the pre-training weight of yolov5-l when running the experiment?

Yes, we use the pre-training weight during the training process.

**Comment by cccssyyy** [comment]
请问最后会打印两个模型的结果，一个是最后一次训练，一个是最好的一次训练吗
